themes/hugo-bulma-blocks-theme/layouts/shortcodes/rich-box-start.html

```diff
@@ -0,0 +1,8 @@
+{{ $icon := .Get "icon" | default ""}}
+<div class="container rich {{ .Get "layout" }}">
```

Comment:
It will be a little better to use more obvious naming, layout -> layoutClass
theme -> themeClass